sites/default/settings.php

```diff
@@ -0,0 +1,243 @@
+    // follow the https://www.uoguelph.ca/sitename pattern, override the reverse proxy
+    // domain and path here.
+    $proxy_conf = array(
+      'a11yportal' => array('domain' => 'wellness.uoguelph.ca', 'path' => '/accessibility/portal'),
+      'accessibilityweb' => array('path' => '/accessibility/web'),
+      'ahl1' => array('path' => '/ahl'),
+      'alliance1' => array('path' => '/alliance'),
+      'arts1' => array('path' => '/arts'),
+      'bioinformatics1' => array('path' => '/bioinformatics'),
+      'business2' => array('path' => '/lang'),
+      'cbspurchasing' => array('path' => '/cbs/purchasing'),
+      'ccs1' => array('path' => '/ccs'),
+      'ccsforms' => array('path' => '/ccs/forms'),
+      'cecs' => array('domain' => 'www.recruitguelph.ca'),
+      'cio1' => array('path' => '/cio'),
+      'cip1' => array('path' => '/cip'),
+      'counselling' => array('domain' => 'wellness.uoguelph.ca'),
+      'cps1' => array('path' => '/cps'),
+      'dhr1' => array('path' => '/diversity-human-rights'),
+      'dunfield' => array('path' => '/crc/dunfield'),
+      'engineering1' => array('path' => '/engineering'),
+      'finance1' => array('path' => '/finance'),
+      'fire1' => array('path' => '/fire'),
+      'fsadmin' => array('path' => '/foodscience/fsadmin'),
+      'fys' => array('path' => '/vpacademic/fys'),
+      'graduatestudies' => array('domain' => 'graduatestudies.uoguelph.ca'),
+      'graduatestudiesforms' => array('path' => '/graduatestudies/forms2'),
+      'gsa1' => array('path' => '/gsa'),
+      'healthservices' => array('domain' => 'wellness.uoguelph.ca', 'path' => '/health'),
+      'hpc1' => array('domain' => 'wellness.uoguelph.ca', 'path' => '/hpc'),
+      'hr1' => array('path' => '/hr'),
+      'ib2' => array('path' => '/ib'),
+      'international1' => array('path' => '/international'),
+      'iqap' => array('path' => '/vpacademic/iqap'),
+      'lvpo' => array('path' => '/finance/departments-services/purchasing-services/lvpo-forms'),
+      'management1' => array('path' => '/management'),
+      'mcs1' => array('path' => '/mcs'),
+      'oac1' => array('path' => '/oac'),
+      'police1' => array('path' => '/police'),
+      'ppp1' => array('path' => '/vpacademic/ppp'),
+      'psa1' => array('path' => '/psa'),
+      'psychology1' => array('path' => '/psychology'),
+      'realestate1' => array('path' => '/realestate'),
+      'researchinnovation' => array('path' => '/research/innovation'),
+      'sas1' => array('domain' => 'wellness.uoguelph.ca', 'path' => '/accessibility'),
+      'ses1' => array('path' => '/ses'),
+      'sws1' => array('domain' => 'wellness.uoguelph.ca'),
+      'studentaffairs1' => array('path' => '/studentaffairs'),
+      'sustainability1' => array('path' => '/sustainability'),
+      'tutoring1' => array('path' => '/tutoring'),
+      'wellnesseducation' => array('domain' => 'wellness.uoguelph.ca', 'path' => '/education'),
```

Comment:
I think wellness.uoguelph.ca/education is no longer a separate Drupal site.

Reply:
Removed.